### PR TITLE
Update tutorial to v0.40.0

### PR DESCRIPTION
**Pull request by jtlan**

### CODE REVIEW

File: sass/_plottable_examples.scss [hunk 1]
```diff
@@ -34,6 +34,10 @@
 //
 // }
 
+a {
+  color: $COLOR_ORANGE1;
+}
```
Comment (bluong) [inline sass/_plottable_examples.scss]: What is this for?

File: tutorials/index.md [hunk 2]
```diff
@@ -11,9 +11,7 @@ Get started with the Plottable Tutorial: Download the packaged tutorial
 files at <http://plottablejs.org/tutorials.zip>; extract
 it, and each subdirectory corresponds to a different tutorial. You can
 open the \`.html\` files directly in the browser, and modify the
-associated \`.js\` files with your favorite text editor. Alternatively,
-you can clone the entire repository from
-<https://github.com/palantir/plottable>.
+associated \`.js\` files with your favorite text editor.
```
Comment (bluong) [inline tutorials/index.md]: I might retract on my previous statment that the tutorials should not be in the plottable repo.  If the user wants to retrieve these handy tutorials as examples, how would they retreive them?

File: tutorials/index.md [hunk 3]
```diff
@@ -181,8 +179,20 @@ function makeBasicChart() {
...
   plot.addDataset(xyData);
+
+  function getXDataValue(d) {
+    return d.x;
+  }
+  plot.project("x", getXDataValue, xScale);
+  function getYDataValue(d) {
+    return d.y;
+  }
+  plot.project("y", getYDataValue, yScale);
```
Comment (bluong) [inline tutorials/index.md]: Doesn't matter too much, but it's a matter of presentation / idiomaticness.  Do we want to inline the function?

File: tutorials/index.md (hunk 2 above)
Comment (jtlan) [inline tutorials/index.md]: There's a .zip in linked from the Tutorials page. We could keep the tutorials in the Plottable repo, but then how do we synchronize them?

**Comment by bluong** [comment]
Just a reminder to update the version on the homepage (I forgot if I said this earlier)

File: sass/_plottable_examples.scss (hunk 1 above)
Comment (jtlan) [inline sass/_plottable_examples.scss]: From the last change I made. It didn't make it to *master* somehow.

File: tutorials/index.md (hunk 2 above)
Comment (bluong) [inline tutorials/index.md]: So I guess my nervousness is surrounding the fear that being able to download the tutorials is not visible.  That said, I think the synchronization issue is more critical, and thus we should just keep it as it is.  If we think about a reorganizing the site, however, we should think about how tbest to let the users know that these examples can be downloaded.

**Comment by jtlan** [comment]
>Just a reminder to update the version on the homepage (I forgot if I said this earlier)

See the last commit (5f39e77).

**Pull request merged**